src/DatePicker/DateDisplay.js

```diff
@@ -47,7 +47,7 @@ function getStyles(props, context, state) {
       marginBottom: 10,
     },
     yearTitle: {
-      cursor: (!selectedYear && !props.disableYearSelection) ? 'pointer' : 'default',
+      cursor: props.disableYearSelection ? 'not-allowed' : (!selectedYear ? 'pointer' : 'default'),
```

Comment:
Changing the `disableYearSelection` property doesn't change the styling of the element.
Hence, I don't think that we should change the `cursor`.
Why should we use the `not-allowed`? I think that we should us it to `emphasize` the *disabled* state.
So when should we use `not-allowed`? I would say as soon as we change the default style.